src/Kanban/index.js

```diff
@@ -233,7 +225,7 @@ class Kanban extends PureComponent {
     return findItemIndex(this.state.lists, itemId);
   }
 
-  renderList({ columnIndex, key, style }) {
+  renderList({ columnIndex, rowIndex, key, style, parent }) {
```

Comment:
for this line, what does the rowIndex and parent help with?

Reply:
Also the CellMeasurer surrounding the SortableList seems optional to me.

I updated it because it existed there already and there was a comment about how it was supposed to fix something related to the scroll bar. However I did not notice any differences after the update with or without it after I'd completed it (the bar is in a slightly offset position now which may be a minor annoyance)

The CellMeasurer surrounding SortableItem now serves the purpose that SortableList did. It might be able to let us create task-lists with dynamic heights (for the sticky footers) but I think it would be better not to since the larger drop zone is convenient and have dynamic columns and dynamic rows could be a performance hit.